recipes/slim-trees/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "slim-trees" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/slim-trees-{{ version }}.tar.gz
+  sha256: 5d148b0d4d0cac79e7a9156a552f69ead2a4afc7d69ffe18889f2236e6355ea2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - numpy
+
+test:
+  imports:
+    - slim_trees
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/pavelzw/slim-trees
```

Comment:
Just FYI, this link currently results in a 404.

Reply:
Oops, fixed that 😅